.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.png filter=lfs diff=lfs merge=lfs -text
+*.tif filter=lfs diff=lfs merge=lfs -text
+*.tiff filter=lfs diff=lfs merge=lfs -text
```

Comment:
How large are these datasets?

If they're small, then they can be put in this repo without LFS, and if they're large, they probably belong on figshare (with download instructions).

I would prefer for these sorts of side projects, which are less likely to be used often, to not add too much bulk to our LFS server

Reply:
I will remove these file from the lfs and purge the git history of the lfs files in this repo!